platforms/web/src/containers/AppRoutes/AppRoutes.tsx

```diff
@@ -24,6 +24,7 @@ export default function AppRoutes() {
         <Route element={<Layout />} errorElement={<RootErrorPage />}>
           <Route index element={<Home />} />
           <Route path={PATH_PLAYLIST} element={<PlaylistScreenRouter />} />
+          <Route path={PATH_CONTENT_LIST} element={<PlaylistScreenRouter />} />
```

Comment:
I would suggest we either use @ChristiaanScheermeijer 's suggestion to have the `type` as a param on PlaylistScreenRouter or alternatively, use a regex for the path and share one <Route> for playlist and content list. I think the former is preferred, because you don't need the extra path match checking inside PlaylistScreenRouter.